src/Summary/SummaryForm.js

```diff
@@ -47,9 +53,24 @@ class SummaryForm extends Component {
         </Col>
       </Row>
     );
-  }
+  };
+
+  hideVendorUncheckConfirm = () => {
+    this.setState({ isVendorUncheckConfirm: false });
+    this.props.dispatchChange('isVendor', true);
+  };
+
+  handleVendorUncheck = () => this.setState({ isVendorUncheckConfirm: false });
```

Comment:
I think you might miss the point of ticket here. Check the message to user - 

> "WARNING All Vendor information, Vendor terms, EDI information, and Accounts will be deleted from this record."

I think you have to remove all of this information from form values

Reply:
> I think you might miss the point of ticket here. Check the message to user -
> 
> > "WARNING All Vendor information, Vendor terms, EDI information, and Accounts will be deleted from this record."
> 
> I think you have to remove all of this information from form values

You think I need to delete all information after confirmation? not after saving?

Reply:
yes, after the confirmation. It's described in ticket's description

> Scenario 4
When the user clicks Confirm
And delete the existing information in the suppressed accordions from the UI and the backend